src/field_theory/splitting_field.lean

```diff
@@ -200,6 +200,29 @@ begin
   exact splits_of_splits_id _ h
 end
 
+theorem exists_splits (f : polynomial α) :
+  ∃ (β : Type u) [field β], by exactI ∃ i : α →+* β, splits i f :=
```

Comment:
This might as well be a `noncomputable` def that chooses the field.

Reply:
But that field isn't small enough...

Reply:
Yeah, you also need to define the map into any field where the polynomial splits, I'm guessing that's not that hard to add to this proof.

Reply:
It's actually a bit difficult (`β` is not the splitting field!), so let's leave it for the next PR.

Reply:
@kckennylau But Chris is suggesting that we change the statement to include the fact that every field that splits the polynomial will receive an algebra hom from `beta`.
Of course this still needs some supporting stuff on algebras and fields that isn't currently in mathlib, but I agree that maybe (after that supporting stuff is added) this proof doesn't get that much harder/longer.
So maybe this PR should wait for a couple of days until after the algebra/field PR?